src/server/saved_objects/service/lib/repository_provider.test.js

```diff
@@ -42,6 +42,7 @@ test('creates a valid Repository', async () => {
   const schema = Symbol();
   const serializer = Symbol();
   const onBeforeWrite = Symbol();
+  const includeHiddenTypes = Array();
```

Comment:
`expect(...).toHaveBeenCalledWith()` doesn't do strict equality on object arguments (or their properties) which is why we use `Symbol()` for the other options, to make sure the exact, unmodified, version of the option is being passed to the `SavedObjectsRepository`. Can use use a `Symbol` for `includeHiddenTypes` too?

Reply:
Symbol() !== Array() so I get an error with the test when I try to use symbol.

Reply:
But I'm not sure if  toHaveBeenCalled with `Array()` does what we want or not.